pom.xml

```diff
@@ -18,6 +18,27 @@
 		</license>
   </licenses>
 
+  <developers>
+    <developer>
+      <id>climbage</id>
+      <name>Michael Park</name>
+      <organization>Esri</organization>
+      <organizationUrl>http://www.esri.com</organizationUrl>
+      <roles>
+	<role>developer</role>
+      </roles>
+    </developer>
+    <developer>
+      <id>randallwhitman</id>
+      <name>Randall Whitman</name>
+      <organization>Esri</organization>
+      <organizationUrl>http://www.esri.com</organizationUrl>
+      <roles>
+	<role>developer</role>
+      </roles>
+    </developer>
+  </developers>
```

Comment:
[Required for Central Repository](http://central.sonatype.org/pages/requirements.html).